app/models/concerns/fog_extensions/google/server.rb

```diff
@@ -12,12 +12,20 @@ def flavors
       end
 
       def image_id
-        image_name unless disks.nil?
+        image_name unless disks.empty?
```

Comment:
use #blank?
empty is (by default) defined in Enumerable.
nil.empty? is defined in ActiveSupport 3.2, but it might get deprecated.